### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Create New file_High Contrast: "Kind" drop-down is not adopting high contrast black mode

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4296]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS4.3.1; #HighContrast ; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2;

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**MAS Violated:** MAS 4.3.1

**Repro Steps:**
1.Apply high contrast black mode. Open Java EE - Eclipse application
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "File > New > Project" in the navigation bar and press enter
4.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
5.Tab till "Next" button and press enter 
6.Enter all the required details, tab till "Next" button and press enter
7.Tab till "Module Dependencies" tab and press enter
8.Tab till "Finish" button and press enter
9.In Project Explorer, expand the created Scala project right-click src, New-->Other and press enter
10.Tab till Scala Wizards, expand and select 'Scala Object' and press next
11.Check if the "Kind" drop-down is adopting high contrast black mode

**Actual Result:**
"Kind" drop-down is not adopting high contrast black mode

**Expected Result:**
"Kind" drop-down should adopt high contrast black mode

**User impact:**
Users with visual impairments will not be able to access the drop-down

**MAS Reference:**
[MAS 4.3.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B3870F7C4-7C43-43C6-A9F7-FD73D800BFD0%7D&file=MAS%204.3.1%20%E2%80%93%20No%20Disruption%20of%20Accessibility%20Features.docx&action=default&mobileredirect=true)

![Bug_4296 _Java Tooling_Create new File_HCB Disruption](https://user-images.githubusercontent.com/41108985/79102077-a75a6e80-7d87-11ea-9091-6f7b6dc4c221.png)
![Bug_4296 _Java Tooling_Create new File_List_HCB Disruption](https://user-images.githubusercontent.com/41108985/79102104-b04b4000-7d87-11ea-888c-d89a929124b1.png)

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Scala Plugin. You can find the same dialog in New Project > Scala Wizards > Scala Project. So it's out of the scope of our toolkit. Please find the screenshot of Eclipse\ScalaPlugin without our toolkit.
![image](https://user-images.githubusercontent.com/36756024/81460357-80048f00-91d7-11ea-834d-ad73a734c67f.png)

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Important. Users with visual impairments will not be easy to know the items in the drop-down when create a scala object.
+ Quality of experience: Manageable. Narrator could read the selected item in the drop-down list.
+ Alternate solutions: User could use narrator to read the selected item.
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

External issues tracked in: https://bugs.eclipse.org/bugs/show_bug.cgi?id=562985

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749647
